### ISSUE TITLE
Unfollow all / close account

### ISSUE THREAD

**Issue by tuukkamustonen** [CodeRelease/CodeRelease#7]
So I clicked "Subscribe to Stars" button, and it added all repos that I have starred.

Now I want to disable/remove each one of them. How? I disabled that toggle, but added repositories are still there - do I have to disable them one-by-one?

**Comment by dayfuaim** [comment]
Yes, I have almost the same question: How to remove subscription from list?
